.github/workflows/grist-deployment-tests.yml

```diff
@@ -9,13 +9,25 @@ on:
   pull_request:
     branches:
       - main
+    paths:
+      - grist-deployment-tests/
```

Comment:
What does this bring please?

(you may answer "RTFM", that would be a good answer)